person.rb

```diff
@@ -1,4 +1,9 @@
+require './nameable'
+require './capitalize_decorator'
+require './trimmer_decorator'
+
 class Person
```

Comment:
- [ ] Kindly make sure that your `Person` class inherits from `Nameable` as per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md#:~:text=Make%20sure%20that%20your%20Person%20class%20inherits%20from%20Nameable)